# Represents a named point cloud for an object or a surface.

string name
sensor_msgs/PointCloud2 cloud